kotlin/change-atmosphere-effect/README.md

```diff
@@ -1,34 +1,36 @@
 # Change atmosphere effect
-Change the appearance of the atmosphere in a scene.
 
-![Change Atmosphere Effect App](change-atmosphere-effect.png)
+Changes the appearance of the atmosphere in a scene.
+
+![Image of change atmosphere effect](change-atmosphere-effect.png)
 
 ## Use case
 
 Atmospheric effect can be used to make the scene view look more realistic.
 
 ## How to use the sample
 
-Select one of the three available atmosphere effects. The sky will change to display the selected atmosphere effect. 
+Select one of the three available atmosphere effects. The sky will change to display the selected atmosphere effect.
 
 ## How it works
-To change the atmosphere effect:
 
 1. Create an `ArcGISScene` and display it in a `SceneView`.
-2. Change the atmosphere effect with `sceneView.atmosphereEffect(AtmosphereEffect)`.
+2. Change the atmosphere effect with `sceneView.atmosphereEffect`.
 
 ## Relevant API
 
-* ArcGISScene
-* AtmosphereEffect
-* SceneView
+- ArcGISScene
```

Comment:
I'm not sure if it makes a difference tbh, but we usually use "*" here rather than "-"
```suggestion
* ArcGISScene
```